### ISSUE TITLE
get_nodes(id = XXXX, files = TRUE) returns "osfstorage"

### ISSUE THREAD

**Issue by McKayMDavis** [CenterForOpenScience/osfr#57]
I noticed this issue as I was attempting to get some WaterButler links for files. I would think that running `get_nodes(id = XXXX, files = TRUE)` should return all files within the parent node rather than just info on `osfstorage`. I have fixed this issue on a forked version of the package and would be happy to share the simple fix.

**Comment by bgrich** [comment]
You are absolutely right. As it is currently written, `get_nodes()` lists the file providers and not the actual files. Getting to the files in osfstorage requires going down one more level. To return the files themselves will require specifying the storage provider, whether it is osfstorage or some other provider. Unfortunately, `get_nodes()` will still not return all of the files if they are within sub-folders. 

Feel free to submit a pull request with your changes!
